Update nested-virtualization.md

andrewallen opened the pull request:
Changed New-VMSwitch -SwitchName to New-VMSwitch -Name as per correct syntax. https://docs.microsoft.com/en-us/powershell/module/hyper-v/new-vmswitch?view=win10-ps

azurecla commented:
Hi __@nnasaki__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, AZPRBOT;

BryanLa commented:
Hey @nnasaki - thanks very much for spotting this and making the contribution! I've merged your changes.

andrewallen committed:
Update nested-virtualization.md

Changed New-VMSwitch -SwitchName to New-VMSwitch -Name as per correct syntax. https://docs.microsoft.com/en-us/powershell/module/hyper-v/new-vmswitch?view=win10-ps

andrewallen committed:
Merge pull request #1 from andrewallen/andrewallen-patch-1

Update nested-virtualization.md
